core/templates/domain/story/story-update.service.spec.ts

```diff
@@ -477,10 +477,10 @@ describe('Story update service', function() {
 
     // Adding an already existing exploration in the story should throw an
     // error.
-    expect(function() {
-      StoryUpdateService.setStoryNodeExplorationId(
-        _sampleStory, 'node_1', 'exp_1');
-    }).toThrow();
+    // expect(function() {
```

Comment:
Let's remove this if it is no longer relevant.

Reply:
Oh, this was accidental, will revert it.

Reply:
Done